classification/views/exports/classification_export_formatter_csv.py

```diff
@@ -44,11 +44,14 @@ class FormatDetailsCSV:
 
     html_handling: CSVCellFormatting = CSVCellFormatting.PURE_TEXT
 
+    include_discordances: bool = True
+
     @staticmethod
     def from_request(request: HttpRequest) -> 'FormatDetailsCSV':
         pretty = request.query_params.get('value_format') == 'labels'
         include_explains = request.query_params.get('include_explains') == 'true'
         exclude_transient = request.query_params.get('exclude_transient') == 'true'
+        include_discordances = request.query_params.get('include_discordances') == 'true'
```

Comment:
I know we jumped back and fourth on if this should be include/exclude, but as the default is for it to be included, we should change it back to "exclude_discordances".
That way, unless it's specified, discordance information will be included (which is what we want, excluding it should be a purposeful descision)